### PR TITLE
[devtools] Format Devtools ESLint config

### PR DESCRIPTION
**Pull request by Alonski**
### Description

Format Devtools ESLint config

### CODE REVIEW

File: packages/devtools/.eslintrc.js [hunk 1]
```diff
@@ -1,185 +1,185 @@
...
-    "number",
-    "String",
-    "string",
-    "Boolean",
-    "boolean",
-    "Undefined"
+  'arrow-body-style': 'error',
+  'arrow-parens': ['off', 'as-needed'],
+  camelcase: 'error',
+  complexity: 'off',
```
Comment (geoknee) [inline packages/devtools/.eslintrc.js]: Is now a good time to fix the `'key':` vs `key:` discrepancy?

Comment (Alonski) [reply packages/devtools/.eslintrc.js]: JavaScript allows us to define keys without quotes surrounding them if they are single words.
So `key: "something"` is valid while `this-key: "something"` is not.
The valid way is `"this-key": "something"`

Comment (geoknee) [reply packages/devtools/.eslintrc.js]: ...so do you think we should have all keys in double quotes? That way all keys have the same format.

Comment (Alonski) [reply packages/devtools/.eslintrc.js]: It is formatted automatically by Prettier. I don't think there is a config for it.
Take a look at this discussion:
https://github.com/prettier/prettier/issues/838

Comment (geoknee) [reply packages/devtools/.eslintrc.js]: Approved since this issue is not caused by this PR.

**Pull request merged**